nilearn/plotting/surf_plotting.py

```diff
@@ -460,6 +462,12 @@ def _compute_facecolors_matplotlib(bg_map, faces, n_vertices,
 
     if darkness is not None:
         bg_faces *= darkness
+        warn(
+            (
+                "The `darkness` parameter will be deprecated in release 0.13. "
+                "We recommend setting `darkness` to None"
+            )
+        )
```

Comment:
This looks good. I would specify the class of warning such as in this example: https://github.com/nilearn/nilearn/blob/e002d2421584ee3f29499f0719a843bdac103054/nilearn/glm/first_level/first_level.py#L933-L936

And then change the class also in the test